turbinia/config/turbinia_config_tmpl.py

```diff
@@ -63,6 +63,9 @@
 # host to properly lock the resource state file.
 RESOURCE_FILE_LOCK = '%s.lock' % RESOURCE_FILE
 
+# For Kubernetes infrastructure. Indicates whether a given pod is set to be deleted.
+TO_BE_DELETED = '%s/turbinia-to-delete.lock' % TMP_DIR
```

Comment:
Can we rename this to something more descriptive?  Maybe `DELETE_WORKER_FILE` or `SCALEDOWN_WORKER_FILE`?  The file path could be a little more descriptive too, maybe `turbinia-worker-delete.lock` or something?  Also, what actually manages this file?

Reply:
The file is only created from when the pod is set to `TERMINATING` and `preStop` gets called and creates a file in the path. It doesn't need the `.lock` extension but added it as it indicates the "worker is locked up" but can change this to `txt` if it makes sense

Reply:
The file is only created from when the pod is set to TERMINATING and preStop gets called and creates a file in the path. It doesn't need the .lock extension but added it as it indicates the "worker is locked up" but can change this to txt if it makes sense. Also updated the variable to `SCALEDOWN_WORKER_FILE` and path name to `turbinia-worker-scaledown.lock`